draft-ietf-jsonpath-base.md

```diff
@@ -416,17 +417,17 @@ be used with a comparable XML document; see also {{xpath-overview}}.
 ## Overview {#synsem-overview}
 
 A JSONPath query is a string which selects zero or more nodes of a piece of JSON.
-A valid query conforms to the ABNF syntax defined by this document.
 
-A query MUST be encoded using UTF-8. To parse a query according to
-the grammar in this document, its UTF-8 form SHOULD first be decoded into
+A query MUST be encoded using UTF-8.
+The grammar for queries given in this document assumes that its UTF-8 form is first be decoded into
```

Comment:
Typo, should be: "... is first decoded into ..."

(Meta comment: I can no longer make suggestions. The icon has disappeared!)

Reply:
To be fixed in next commit.